src/model/output.test.ts

```diff
@@ -6,4 +6,9 @@ describe('Output', () => {
       await expect(Output.setArtifactsPath('')).resolves.not.toThrow();
     });
   });
+  describe('setCoverageResultsPath', () => {
+    it('does not throw', async () => {
+      await expect(Output.setCoverageResultsPath('')).resolves.not.toThrow();
```

Comment:
Lets test with an actual path and not an empty string.

Reply:
I'll add some more test, good point :)

Reply:
Added some tests 😄